src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -61,7 +61,9 @@ Known issues:
     <string name="revanced_enable_old_player_background_summary">Returns the player background to the old style.</string>
     <string name="revanced_enable_old_player_background_title">Enable old player background</string>
     <string name="revanced_enable_old_player_layout_summary">"Returns the player layout to the old style.
-Some features may not work properly in the old player layout."</string>
+Some features may not work properly in the old player layout.
+        
+WARNING: Do not hide like and dislike buttons while this is enabled."</string>
```

Comment:
The Like/Dislike hiding toggle can still be turned on/off, it just doesn't work. Therefore, using "DO NOT" in this case is unreasonable in my opinion

Reply:
But for me it crashes... I thought that this is intended behaviour

Reply:
Hmm so I think you can revert and add back this changes

Reply:
Reverted